patches/server/0766-Revert-MC-4-fix.patch

```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Spottedleaf <spottedleaf@spottedleaf.dev>
+Date: Tue, 22 Dec 2020 12:06:15 -0800
+Subject: [PATCH] Revert MC-4 fix
+
+When messing around with collisions, I ran into problems where
+entity position was off by ULP and that caused clipping problems.
+Now, the collision epsilon is 1.0e-7 to account for those errors.
+
+But this patch is going to cause problems on the order of 1.0e-4.
+
+I do not want to deal with clipping problems. The very fact it works
+shows it's causing the clipping to occur serverside.
+
+diff --git a/src/main/java/net/minecraft/world/entity/Entity.java b/src/main/java/net/minecraft/world/entity/Entity.java
+index 20edcecf422c7feb05070620b8d95546079825b4..cdf8d623c84c8369ee68e56e0925db87eb5476c4 100644
+--- a/src/main/java/net/minecraft/world/entity/Entity.java
++++ b/src/main/java/net/minecraft/world/entity/Entity.java
+@@ -3937,7 +3937,7 @@ public abstract class Entity implements Nameable, EntityAccess, CommandSource, n
+         // Paper end
+         // Paper start - fix MC-4
+         if (this instanceof ItemEntity) {
+-            if (com.destroystokyo.paper.PaperConfig.fixEntityPositionDesync) {
++            if (false && com.destroystokyo.paper.PaperConfig.fixEntityPositionDesync) { // Paper - revert
```

Comment:
Probably don't want to add this patch since it just reverts another Paper patch?